docs/INTRO-cmake.md

```diff
@@ -5,7 +5,12 @@ The easiest way to use SDL is to include it as a subproject in your project.
 
 We'll start by creating a simple project to build and run [hello.c](hello.c)
 
-Create the file CMakeLists.txt
+# Get a copy of the SDL source:
+```sh
+git clone https://github.com/libsdl-org/SDL.git
```

Comment:
We're about to include it in the vendored subdirectory, so you probably want to clone it there instead.

Reply:
Whoops, I did intend for that, I'll fix it!